### DISCUSSION TITLE
ENV Variables and other issues

### DISCUSSION THREAD

**Discussion by choff5507** [coollabsio/coolify#2356] [Q&A]
I am new to Coolify and trying to get V4 up and running. The two issues I am having are this:

![SCR-20240604-ppqp](https://github.com/coollabsio/coolify/assets/19677995/e557d58b-cc31-45b9-95f1-fc630280f97c)

I have no idea what this error means in regards to the ENV variables. It sounds like it's referring to the docker compose file but I am not using one and using Nixpacks to deploy. 

Also, It seems like the container starts and then continues to restart. If i docker logs the container I see it says the following error:

`/bin/bash: -c: option requires an argument`

If it matters I am trying to deploy a Nuxt application. Can anyone give me any insights ?

**Comment by SSardorf** [comment]
Regarding the `/bin/bash: -c: option requires an argument`
I have the same issue. Not sure what's causing it either.

**Comment by SpuriousGer** [comment]
I am getting the same issue. What is this caused by? @andrasbacsai could you please have a look?
It goes away for me if I click the "Static Page" checkbox. But I am deploying a Flask app, not sure if this is actually correct.

**Comment by SSardorf** [comment]
@SpuriousGer @choff5507 
Found the issue.
It happens when your start command does not match what's in your package.json.

So Nixpacks defaults to `npm start`
If you don't have a script in your package.json called `start` - it'll throw `/bin/bash: -c: option requires an argument` repeatedly

**Reply by arshx86** [reply]
Thank you so much. I just missed `"start": "nuxt start",` in `package.json` and added it after seeing your comment and it now works.

**Comment by SpuriousGer** [comment]
That's good to know for node. But I was running a Python Flask application. What is the equivalent?

**Reply by SSardorf** [reply]
> That's good to know for node. But I was running a Python Flask application. What is the equivalent?

Should be `main.py`
Do you have a `main.py` file?

**Reply by SpuriousGer** [reply]
> > That's good to know for node. But I was running a Python Flask application. What is the equivalent?
> 
> Should be `main.py` Do you have a `main.py` file?

No, it's called app.py. What do I need to do?

**Reply by sisou** [reply]
Set a custom start/run command in the coolify app config.
